tests/test_xpub_cmd.py

```diff
@@ -0,0 +1,16 @@
+def test_get_public_key(cmd, button):
```

Comment:
Our get xpub returns the xpub in the change derivation path, right? And is it possible to validate more than just the length of the responses?

Reply:
We return the xpub data on the path passed and since the seed used by the simulator is unknown I don't think we can verify the content of the response.